setup/files/docker-redash-nginx/nginx.conf

```diff
@@ -0,0 +1,20 @@
+events {
+  worker_connections  4096;  ## Default: 1024
+}
+
+http {
+  upstream redashapp {
+    server redash:5000;
+  }
+
+  server {
+    listen  80;
+
+    access_log  /var/log/nginx/log/access.log;
+    error_log  /var/log/nginx/log/error.log;
+
+    location / {
+      proxy_pass  http://redashapp;
```

Comment:
Unless in newer versions of nginx those options are on by default worth adding to the server directive:
```
  gzip on;
  gzip_types *;
  gzip_proxied any;
```

and to the `location /` directive:
```
    proxy_set_header Host $http_host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto $scheme;
```